README.md

```diff
@@ -175,22 +175,14 @@ For Browser Support and the configuration of the UI5 Web Components, please take
 
 #### Internet Explorer 11
 
-`@ui5/webcomponents-react` is supporting all modern major browsers. There is no support for Internet Explorer 11 built in.<br />
-If you want your application to run in IE11, you will have to polyfill some features by importing these polyfills as **first** imports in your `src/index.js`:
-
-```js
-import 'react-app-polyfill/ie11';
-import '@ui5/webcomponents-base/dist/features/browsersupport/IE11WithWebComponentsPolyfill';
-import '@ui5/webcomponents-react-base/polyfill/IE11';
-```
-
-You can install `react-app-polyfill` with the following command:
-
-```bash
-npm install react-app-polyfill --save
-```
-
-In case you are using [`browserslist`](https://github.com/browserslist/browserslist) (which is part of `create-react-app`), please also make sure you have added `IE 11` to your browserslist configuration (can be found in `package.json` if you are using CRA).
+UI5 Web Components are supported by all major modern browsers.
+
+Browser | Supported versions
+--------|---------------------
+Chrome  | Latest
+Firefox | Latest
+Safari  | Latest
+Edge    | Latest
```

Comment:
Should we also include that we support all major mobile browsers?

```suggestion
Edge    | Latest
Opera   | Latest
```

Reply:
I'm not sure about this one to be honest - I've just copied the list from the `@ui5/webcomponents` project to be honest.
Which mobile browsers would you see in this list? Because with Safari and Chrome we should cover most of the iOS/Android devices, right?

Reply:
I think it would be sufficient if we mention that we also include modern mobile browser without adding a browser list.

Or we could remove the list completely, since it should be apparent which browsers are meant.